### DISCUSSION TITLE
Customisation for notifications

### DISCUSSION THREAD

**Discussion by lleewwiiss** [mantinedev/mantine#2802] [Q&A] [answered]
Is it possible to somehow extend the notification component to include other things like links, buttons or images?

The only way I currently see how is by putting a custom component in the icon prop,  basically, I'm trying to replicate the features from web push notifications.

**Comment by rtivital** [comment, marked as answer]
You can put any jsx in `message`: `showNotification({ message: <button>Your button</button> })`

**Reply by lleewwiiss** [reply]
Oh, that makes sense, awesome thank you for your work on such a great package!
